.gitignore

```diff
@@ -11,7 +11,7 @@
 .mtj.tmp/
 
 # Package Files #
-*.jar
+# *.jar
```

Comment:
Shouldn't jar files still be excluded from git commits?

Reply:
Added back in.